config/betterquesting/DefaultQuests/Quests/MultiblockGoals-AAAAAAAAAAAAAAAAAAAADA==/PreciseAssembler-AAAAAAAAAAAAAAAAAAAMEg==.json

```diff
@@ -50,8 +54,12 @@
       "consume:1": 0,
       "requiredItems:9": {
         "0:10": {
-          "id:8": "GoodGenerator:impreciseUnitCasing",
+          "id:8": "betterquesting:placeholder",
```

Comment:
you broke something here

Reply:
ah yep i sure did, is there a way of ignoring that one line change? if not ill just boot up a nightly